test/index.js

```diff
@@ -70,6 +70,27 @@ describe('validations', () => {
     falsy.forEach((item) => assert.isFalse(validations.isEmail(item), `Failed for ${item}`));
   });
 
+  it('isMostLikelyEmail', () => {
+    const truthy = [
+      'a@example.com',
+      'a+b@example.com',
+      'a.b+c@example.com',
+      'a_b.c+d@example.com',
+      'asdasdasd@.',
+    ];
+
+    const falsy = [
+      'a.example.com',
+      '@',
+      'a@b',
+      'a@b.de',
```

Comment:
that isnt allowed?

Reply:
No, because we want to ensure a min length of 7 chars. Behind the '@', there has to be at least two chars.

Reply:
Didn't read the email spec but I still wonder if that's a valid requirement 🤷

Reply:
Yes, since a domain has at least 2 chars, and the part behind the the '@' has to be at least 2 chars. 
1+@+2+.+2 = 7